cmake/Find3dsm.cmake

```diff
@@ -1,50 +1,37 @@
 include(FindPackageHandleStandardArgs)
 
-find_path(3dsm_PATH include/maxapi.h)
+if(WIN32)
+    # Karnage: Be sure to select a 64-bit capable 3ds Max SDK on 64-bit builds.
+    set(_TEST_FILES "include/maxapi.h")
+    if(CMAKE_SIZEOF_VOID_P EQUAL 8)
+        set(_LIBDIR "x64/lib")
+        list(APPEND _TEST_FILES "x64/lib/gup.lib")
+    else()
+        set(_LIBDIR "lib")
+    endif()
 
-find_path(3dsm_INCLUDE_DIR maxapi.h
-    ${3dsm_PATH}/include
-)
+    find_path(3dsm_PATH NAMES ${_TEST_FILES}
+        PATHS "C:/3dsmax7" ENV ADSK_3DSMAX_SDK_2012
```

Comment:
Is this PATHS line really needed?  What are the odds that someone will be using Max 7 AND have it installed at `C:\3dsmax7`?  Also, for the `ENV` line, do we need additional entries for other (marginally) supported versions?  I'd probably just remove this whole line, personally.

Reply:
`C:\3dsmax7` is the default install location for 3dsmax7 (yikes) - and yeah, it looks like each version would have its own `ENV` entry 😞